app/code/core/Mage/Eav/Model/Config.php

```diff
@@ -209,7 +209,7 @@ protected function _loadEntityTypes()
     protected function _loadEntityAttributes($entityType, $storeId)
     {
         // preload attributes in array form to avoid instantiating models for every attribute even if it is never accessed
-        $entityAttributes = $entityType->getAttributeCollection()
+        $entityAttributes = $entityType->newAttributeCollection(null, false)
```

Comment:
```suggestion
        $entityAttributes = $entityType->newAttributeCollection()
```

first param should be null by default, second one doesn't exist if i'm not mistaken